components/Auctions/Auctions.module.css

```diff
@@ -0,0 +1,152 @@
+.auctionPageNavigation {
+  text-align: center;
+}
+
+.mainContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 100px;
+}
+
+.auctionContainer {
+  display: flex;
+  width: 70%;
+  background-color: white;
+  justify-content: space-around;
+  align-items: center;
+  border: 1px solid #f4f4f4;
+  border-radius: 20px;
+  box-shadow: 0 0 15px -7px rgba(0,0,0,.65);
+  margin-bottom: 20px;
+  padding: 50px 0;
+}
+
+.auctionSeller {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.auctionSeller div {
+  border-radius: 50%;
+}
+
+.itemInfo {
+  display: flex;
+  justify-content: center;
+}
+
+.gemImage {
+  margin-top: 20px;
+}
+
+.currentBidPrice {
+  font-weight: 400;
+}
+
+.bidOptions {
+  display: flex;
+  flex-direction: column;
+}
+
+.inputBid {
+  border: 0.5px solid #ccc;
+  border-radius: 5px;
+  padding: 10px;
+  margin-bottom: 15px;
+  font-size: 1rem;
+  width: 150px;
+}
+
+.bidBtn {
+  height: 50px;
+  width: 150px;
+  margin-top: 10px;
+  color: white;
+  border-color: rgba(27, 31, 35, 0.15);
+  border-radius: 6px;
+  background-color: #42a742;
+}
+
+.bidBtn:hover {
+  height: 50px;
+  box-shadow: 0 0 15px -7px rgba(0,0,0,.65);
+  cursor: pointer;
+}
+
+.bidders {
+  display: flex;
+  flex-wrap: wrap;
+  width: 25%;
+}
+
+.bidders div {
+  border-radius: 50%;
+}
+
+.biddersImg {
+  margin-left: 10px;
+}
+
+.biddersImg[data-columns="2"]{
+  width: calc(160px/2);
+}
+.biddersImg[data-columns="3"]{
+  width: calc(160px/3);
+}
+.biddersImg[data-columns="4"]{
+  width: calc(160px/4);
+}
+.biddersImg[data-columns="5"]{
+  width: calc(160px/5);
+}
+.biddersImg[data-columns="6"]{
+  width: calc(160px/6);
+}
+.biddersImg[data-columns="7"]{
+  width: calc(160px/7);
+}
```

Comment:
What is this doing?

Reply:
Here we are changing the size of the bidder's image
Size of the bidder's Image-based on the number of columns allowed in the container horizontally.

Reply:
Why not use a declarative class or an inline sizing?